app/views/posts/show.html.erb

```diff
@@ -1 +1,17 @@
-<h1>Here is a post for a given posts:</h1>
+<%= render "shared/header" %>
+  <div class="single-post">
+    <h2><%= @post.title %></h2>
+    <div>
+      <span>Comments: <%= @post.comments_counter %></span>
+      <span>Likes: <%= @post.likes_counter %></span>
+    </div>
+     <p><%= @post.text %></p>
+  </div>
+
+   <h1 class="comments-header">All comments</h1>
+  <div class="post-comments">
+  <% @post.five_most_recent_comments.each do |comment| %>
+      <h4>Username: <%= comment.user.name %> - <span> <%= comment.text %> </span> </h4>
+  <% end %>
+  </div>
```

Comment:
- [ ] Great work attempting to make the changes suggested by the previous reviewer. As earlier mentioned by the previous reviewer, the `five_most_recent_comments` should be used in the Posts#Index page and not the Posts#Show page. Kindly use the method on the correct page. You can use this wireframe as a guide to know when to use it. 👍 💻 

![image](https://github.com/Megagig/Blog-App/assets/77106962/3995d50c-ce10-42db-b88d-0755ef152e9f)

#### N.B: Kindly test the application to ensure that you implemented all changes correctly. 👍 😎